### ISSUE TITLE
Separate application id for debug version

### ISSUE THREAD

**Issue by simontb** [ligi/PassAndroid#167]
As a contributor I would like to be able to have the release version installed on my device alongside with the work I currently do in the debug version.

With gradle this is not a big deal: https://developer.android.com/studio/build/application-id.html#change_the_application_id_for_build_variants

If you agree with my idea, I'm happy to open a PR.

**Comment by ligi** [comment]
Thanks - but I would not want this as a default behavior - was always leading to confusion and WTF moments that I like to avoid. But if you think it makes your life easier - perhaps add it as an option behind a flag?

**Comment by simontb** [comment]
Something like this?
 https://github.com/simontb/PassAndroid/commit/a976a2f44bed9fd0b40bc63a0b85a3406bd5dbb3

It works and Gradle compiles the correct .apk. However, Android Studio does not recognize that it has to start the .debug application and starts the productive one instead. This does not happen, when I put the applicationIdSuffix in the debug build type directly and not in a if-branch.

I think I will work with this locally and apply assume-unchanged, because I don't want to introduce a build flavor

**Comment by ligi** [comment]
Yes - something like this - but I do not think debug_authority_fileprovider is needed - AFAIK you can just use ${applicationId} in the Manifest there. Also the name can change - so we can completely get rid of the else branch
